stable/mission-control/README.md

```diff
@@ -461,8 +461,8 @@ The following table lists the configurable parameters of the mission-control cha
 | `insightServer.name`                         | Insight Server name                             | `insight-server`                      |
 | `insightServer.image.repository`             | Container image                                 | `docker.bintray.io/jfrog/insight-server`|
 | `insightServer.image.version`                | Container image tag                             | `.Chart.AppVersion`                   |
-| `insightServer.externalHttpPort`             | Insight Server service external port            | `8082`                                |
-| `insightServer.internalHttpPort`             | Insight Server service internal port            | `8082`                                |
+| `insightServer.externalPort`                 | Insight Server service external port            | `8082`                                |
```

Comment:
This is breaking. You need to say that in the CHANGELOG and bump the version to more than a patch

Reply:
Done

Reply:
Done